package.json

```
@@ -49,40 +50,29 @@
   },
   "homepage": "https://github.com/react-dropzone/attr-accept#readme",
   "devDependencies": {
-    "@commitlint/cli": "^8.3.6",
+    "@commitlint/cli": "^19.5.0",
+    "@commitlint/config-conventional": "^19.5.0",
     "@size-limit/esbuild": "^11.1.6",
     "@size-limit/esbuild-why": "^11.1.6",
     "@size-limit/preset-small-lib": "^11.1.6",
-    "cz-conventional-changelog": "^1.2.0",
-    "eslint": "^4.6.1",
-    "eslint-config-airbnb": "^15.1.0",
-    "eslint-config-okonet": "^6.1.3",
-    "eslint-config-prettier": "^2.4.0",
-    "eslint-plugin-flowtype": "^2.35.1",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-node": "^5.1.1",
-    "eslint-plugin-prettier": "^2.2.0",
-    "eslint-plugin-react": "^7.3.0",
-    "expect": "^1.20.2",
-    "husky": "^0.14.3",
-    "lint-staged": "^4.1.3",
-    "prettier": "^1.6.1",
+    "eslint": "^9.12.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "husky": "^9.1.6",
+    "lint-staged": "^15.2.10",
+    "prettier": "^3.3.3",
     "semantic-release": "^24.2.0"
   },
   "engines": {
     "node": ">=18"
   },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
-  },
   "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "git add"
-    ]
+    "*.{js,ts}": [
+      "prettier --write",
+      "eslint --fix"
+    ],
+    "!(*.js|*.ts)": "prettier --write --ignore-unknown"
```

Currently we don't have ESLint configured to handle TypeScript in this repo, as it is written purely in JavaScript. So let's omit this until we decide to adopt TypeScript.
```suggestion
    "*.js": [
      "prettier --write",
      "eslint --fix"
    ],
    "!(*.js)": "prettier --write --ignore-unknown"
```

---

Yeah, that's just an artifact of the copy paste. I'll remove the unnecessary `*.ts` pattern.